docs/extensions/accounts/releasing.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Releasing"
+weight: 70
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/extensions/accounts
+geekdocFilePath: releasing.md
+---
+
+{{< toc >}}
+
+## Releasing
+
+After adding changes to the Accounts package within oCIS and testing them locally, you want to update the compiled assets to the oCIS binary. 
+
+To achieve this, you have run a Go command and add the results to your PR.
+
+### Package Hierarchy
+
+- [ocis](https://githug.com/owncloud/ocis)
+    - [ocis-accounts](https://github.com/owncloud/ocis/tree/master/accounts)
+
+#### Updating ocis-accounts
+
+1. Make sure you are inside the [ocis repository](https://github.com/owncloud/ocis) and on your feature branch
+2. Change into accounts' asset package folder via `cd accounts/pkg/assets`
```

Comment:
running 'make generate' in the root of the oCIS repository generates these assets for all oCIS extension and you also can invoke it inside the folder of an extension to just run in for a single extension.

It is already described here, but it seems to be not accessible enough / in an place where people look for it: https://owncloud.dev/ocis/development/build/#build-the-ocis-binary

Reply:
Should we keep both texts since both are (technically) correct? I wrote those with the idea in mind that somebody only touches the `accounts`/`settings` extension and finds all the necessary information in one place :)

Reply:
I see it as a very good addition but would like to advice people only to use 'make generate' because it acts as abstraction an d also does more than just 'go generate'.

Reply:
Updated the PR with a suggestion!

Reply:
Okay @wkloucek I just ran `make generate` on master and ended up with five `embed.go` files that vary from the ones in version control 🥴 and now I'm confused!

Reply:
@wkloucek updated the guides and reduced them to the `make generate` cmd